### PR TITLE
update home content for August 9th

### LINKED ISSUE #5808: 9 August homepage content update
For the **9 August** push, please update the homepage to reflect the programming detailed [here](https://docs.google.com/document/d/1g4RuGKW__eTHNd277rKPMM46ygonXzW68jSs2OXEkPM/edit#).

@muffinresearch

### PR DESCRIPTION
**Pull request by willdurand**
Fix #5808 

---

I was not brave enough to refactor the `HomeHeroBanner` code but we might improve things by having an array of data instead of just an array of components:

```js
const sections = [
  {
    title: i18n.gettext('addon'),
    subtitle: i18n.gettext('super addon'),
    slug: 'addon-slug',
  },
];
```

That would at least help with the `key` prop that has to be updated.

### CODE REVIEW

File: src/amo/components/Home/index.js [hunk 1]
```diff
@@ -26,10 +26,9 @@ import Icon from 'ui/components/Icon';
 import './styles.scss';
 
 export const FEATURED_COLLECTIONS = [
+  { slug: 'refined-reading', username: 'mozilla' },
```
Comment (rebmullin) [inline src/amo/components/Home/index.js]: for some reason I don't see this collection showing up on the frontend...do you?

Comment (rebmullin) [reply src/amo/components/Home/index.js]: oh I guess b/c a version of this collection is not on dev...

**Comment by rebmullin** [comment]
> I was not brave enough to refactor the HomeHeroBanner code but we might improve things by having an array of data instead of just an array of components:

I think this is a great idea. It seems risky to you tho?

File: src/amo/components/Home/index.js (hunk 1 above)
Comment (rebmullin) [reply src/amo/components/Home/index.js]: Just wondering - do you have permissions to create this on dev?

Comment (willdurand) [reply src/amo/components/Home/index.js]: no 😥 can you or @kumar303 create it by chance?

Comment (willdurand) [reply src/amo/components/Home/index.js]: @wagnerand created the collection 🎉

Comment (willdurand) [reply src/amo/components/Home/index.js]: FTR: we (devs) can create a collection under our own account and ask an admin to transfer this collection to the `mozilla` account. We could also setup a VPN and access the admin page :)

File: src/amo/components/HomeHeroBanner/index.js [hunk 2]
```diff
@@ -110,39 +110,108 @@ export class HomeHeroBannerBase extends React.Component<Props> {
...
+        <p>
+          {i18n.gettext(
+            'Drag text, images, or links to perform actions like copy, open, search, and more',
+          )}
+        </p>
+      </HeroSection>,
+      <HeroSection key="hero-27" linkTo="/addon/behind-the-overlay-revival/">
+        <h3>{i18n.gettext('Behind The Overlay Revival')}</h3>
+
+        <p>{i18n.gettext('Click a button to close annoying pop-ups')}</p>
```
Comment (yfdyh000) [inline src/amo/components/HomeHeroBanner/index.js]: It is easy to be misunderstood as closing the pop-up window instead of the overlay popup.

**Pull request merged**